### ISSUE TITLE
Floating scrollbars in lollypop

### ISSUE THREAD

**Issue by kolgza** [pop-os/gtk-theme#318]
**Distribution (run `cat /etc/os-release`):**
<pre><font color="#4E9A06"><b>colin@colin-pc</b></font>:<font color="#3465A4"><b>~/Desktop</b></font>$ cat /etc/os-release
NAME=&quot;Pop!_OS&quot;
VERSION=&quot;19.04&quot;
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME=&quot;Pop!_OS 19.04&quot;
VERSION_ID=&quot;19.04&quot;
HOME_URL=&quot;https://system76.com/pop&quot;
SUPPORT_URL=&quot;http://support.system76.com&quot;
BUG_REPORT_URL=&quot;https://github.com/pop-os/pop/issues&quot;
PRIVACY_POLICY_URL=&quot;https://system76.com/privacy&quot;
VERSION_CODENAME=disco
UBUNTU_CODENAME=disco
</pre>
**Gtk+ 3.0 version - (run ```apt policy libgtk-3-0```):**
<pre><font color="#4E9A06"><b>colin@colin-pc</b></font>:<font color="#3465A4"><b>~</b></font>$ apt policy libgtk-3-0
libgtk-3-0:
  Installed: 3.24.8-1ubuntu1
  Candidate: 3.24.8-1ubuntu1
  Version table:
 *** 3.24.8-1ubuntu1 500
        500 http://us.archive.ubuntu.com/ubuntu disco/main amd64 Packages
        100 /var/lib/dpkg/status
</pre>

**Desktop environment:**
GNOME 3.32

**Pop version (run ```apt policy pop-gtk-theme```):**
<pre><font color="#4E9A06"><b>colin@colin-pc</b></font>:<font color="#3465A4"><b>~</b></font>$ apt policy pop-gtk-theme
pop-gtk-theme:
  Installed: 4.1.4~1560290633~19.04~f75e86a
  Candidate: 4.1.4~1560290633~19.04~f75e86a
  Version table:
 *** 4.1.4~1560290633~19.04~f75e86a 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu disco/main amd64 Packages
       1001 http://ppa.launchpad.net/system76/pop/ubuntu disco/main i386 Packages
        100 /var/lib/dpkg/status
<font color="#4E9A06"><b>colin@colin-pc</b></font>:<font color="#3465A4"><b>~</b></font>$ flatpak info org.gtk.Gtk3theme.Pop

Pop Gtk theme - Pop Gtk theme used by System76

<b>          ID:</b> org.gtk.Gtk3theme.Pop
<b>         Ref:</b> runtime/org.gtk.Gtk3theme.Pop/x86_64/3.22
<b>        Arch:</b> x86_64
<b>      Branch:</b> 3.22
<b>      Origin:</b> flathub
<b>  Collection:</b> org.flathub.Stable
<b>Installation:</b> system
<b>   Installed:</b> 390.7 kB

<b>      Commit:</b> 6154d9636034f0866dc7a2a08815931c641298ccfe1f5dd180875f4a4cb20f03
<b>      Parent:</b> e653cbdae328be72a95013ad5f6733e7b0db9990bd358404e62c9b64a9aa972c
<b>     Subject:</b> Update to 4.1.3 (6a689c1b)
<b>        Date:</b> 2019-03-30 18:57:42 +0000
</pre>

**Related Application and/or Shell theming (or name of widget)**
Lollypop
<pre><font color="#4E9A06"><b>colin@colin-pc</b></font>:<font color="#3465A4"><b>~</b></font>$ flatpak info org.gnome.Lollypop

Lollypop - Play and organize your music collection

<b>          ID:</b> org.gnome.Lollypop
<b>         Ref:</b> app/org.gnome.Lollypop/x86_64/stable
<b>        Arch:</b> x86_64
<b>      Branch:</b> stable
<b>     Version:</b> 1.1.4
<b>     License:</b> GPL-3.0+
<b>      Origin:</b> flathub
<b>  Collection:</b> org.flathub.Stable
<b>Installation:</b> system
<b>   Installed:</b> 52.6 MB
<b>     Runtime:</b> org.gnome.Platform/x86_64/3.32
<b>         Sdk:</b> org.gnome.Sdk/x86_64/3.32

<b>      Commit:</b> 78ab7a1eaab9497f781833eab509ab8c244b2faa6e365ddd1a9d8d4eb708b333
<b>      Parent:</b> 1726d072441552b238b02df8ef8b45d749e38e0d3ad7321ce3fe67a6bef7e9f0
<b>     Subject:</b> SHA256 was indeed not correct (c1f47973)
<b>        Date:</b> 2019-08-19 19:31:39 +0000
</pre>

**Actual issue**
The scroll bar creates a gap in the UI
![Screenshot from 2019-08-21 13-08-23](https://user-images.githubusercontent.com/46303203/63453121-f2e59400-c415-11e9-89df-d42e99894c92.png)

**Steps to reproduce**
Install Flathub version of Lollypop and use the Pop theme. This issue should persist with all versions of Lollypop, however.

**Expected behavior**
The background of the alphabetical scroll bar (I'm not sure what it's really called) should be transparent. 

**Behavior in Adwaita**
![Screenshot from 2019-08-21 13-22-37](https://user-images.githubusercontent.com/46303203/63453491-c716de00-c416-11e9-8d04-dee66478b40d.png)

**Comment by isantop** [comment]
I can't see how to get an alphabetical scrollbar. The list view mode button is disabled in all views. 

This is the Lollypop version I'm using:

```
flatpak info org.gnome.Lollypop

Lollypop - Play and organize your music collection

          ID: org.gnome.Lollypop
         Ref: app/org.gnome.Lollypop/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 1.1.4
     License: GPL-3.0+
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 53.0 MB
     Runtime: org.gnome.Platform/x86_64/3.34
         Sdk: org.gnome.Sdk/x86_64/3.34

      Commit: ddb84ee7bf652c577de2f30dfd25c18cad4de08a038be99ec93b82e3295dac52
      Parent: 93a00835afc21a9abb777c205041c43468e9eacb9b1e9ca06928c766796bd1ea
     Subject: 1.1.4.17 has been removed. Reverting to 1.1.4.16 (9ac6ffc1)
        Date: 2019-10-21 08:25:56 +0000
```

**Comment by kolgza** [comment]
Preferences > Items in sidebar > Artists

**Comment by kolgza** [comment]
It was actually fixed with the release of Pop GTK theme 5.0
